src/main/java/edu/hm/hafner/util/StringContainsUtils.java

```diff
@@ -55,7 +54,53 @@ public static boolean containsAnyIgnoreCase(@Nullable final CharSequence input,
         return false;
     }
 
+    /**
+     * Checks if the provided string contains irrespective of case all of the strings in the given array, handling
+     * {@code null} strings. Case-insensitivity is defined as by {@link String#equalsIgnoreCase(String)}.
+     * <p>
+     * A {@code null} {@code cs} CharSequence will return {@code false}. A {@code null} or zero length search array will
+     * return {@code false}.
+     * </p>
+     *
+     * <pre>
+     * StringUtils.containsAny(null, *)            = false
+     * StringUtils.containsAny("", *)              = false
+     * StringUtils.containsAny(*, null)            = false
+     * StringUtils.containsAny(*, [])              = false
+     * StringUtils.containsAny("abcd", "ab", null) = false
+     * StringUtils.containsAny("abcd", "ab", "cd") = true
+     * StringUtils.containsAny("abc", "d", "abc")  = false
+     * StringUtils.containsAny("ABC", "d", "abc")  = false
+     * </pre>
+     *
+     * @param input
+     *        The string to check, may be {@code null}
+     * @param searchTexts
+     *        The strings to search for, may be empty. Individual CharSequences may be null as well.
+     *
+     * @return {@code true} if all of the search CharSequences are found, {@code false} otherwise
+     */
+    public static boolean containsAllIgnoreCase(@Nullable final CharSequence input,
+            @Nullable final String... searchTexts) {
+        if (StringUtils.isEmpty(input)) {
+            return false;
+        }
+        if (searchTexts == null || searchTexts.length == 0) {
+            return false;
+        }
+
+        for (String searchText : searchTexts) {
+            if (!(StringUtils.containsIgnoreCase(input, searchText))) {
+                return false;
+            }
+        }
+        return true;
+    }
+
```

Comment:
Bitte niemals selbst im `master` Branch arbeiten. Immer einen neuen Branch für jedes Feature erstellen, der vom Master-Branch abzweigt. Siehe dazu: [About branches](https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/about-branches)
_Deinen_ Master musst du dann nur noch falls notwendig mit den Änderungen von Herrn Hafners Repository aktualisieren. Siehe dazu: [Syncing a fork](https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/syncing-a-fork)

Reply:
Eine Frage, wenn ich nun mein SetTest verbessere soll ich dann eine neue Pull request erstellen ?

Reply:
Normalerweise nicht. Es sei denn du legst dafür einen neuen Branch an.